config/flags_terms.go

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) 2020 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package config
+
+import "github.com/urfave/cli/v2"
+
+var (
+	// FlagAgreedTermsConditions agree with terms & conditions.
+	FlagAgreedTermsConditions = cli.BoolFlag{
+		Name:  "agreed-terms-and-conditions",
+		Usage: "Agree with terms & conditions for consumer, provider or both depending on the command executed",
+	}
+
+	// FlagTermsConsumerAgree marks that consumer TOS has been agreed.
+	FlagTermsConsumerAgree = cli.BoolFlag{
+		Name:  "terms-agreed-consumer",
+		Usage: "Defines test network configuration",
+	}
+
+	// FlagTermsProviderAgree marks that provider TOS has been agreed.
+	FlagTermsProviderAgree = cli.BoolFlag{
+		Name:  "terms-agreed-provider",
+		Usage: "Defines test network configuration",
+	}
+
+	// FlagTermsVersion is the current agreed TOS version.
+	FlagTermsVersion = cli.StringFlag{
+		Name:  "terms-agreed-version",
+		Usage: "Defines test network configuration",
+	}
```

Comment:
It will be very confusing for users what they need to use to agree. 
What is the difference between `agreed-terms-and-conditions` and `agreed-terms-and-conditions`?